habitat-lab/habitat/gui/replay_gui_app_renderer.py

```diff
@@ -57,11 +76,27 @@ def post_sim_update(self, post_sim_update_dict):
         for keyframe in keyframes:
             self._replay_renderer.set_environment_keyframe(env_index, keyframe)
 
+        if len(keyframes):
+            self._need_render = True
+
         if "debug_images" in post_sim_update_dict:
             self._debug_images = post_sim_update_dict["debug_images"]
 
-        if len(keyframes):
-            self._need_render = True
+        if "info" in post_sim_update_dict:
```

Comment:
given the driver can already print exactly what it wants where it wants, why is this whole `if "info" in post_sim_update_dict` block needed? Can it be removed?

Reply:
Sorry, my bad. This had to be removed.